code/game/machinery/seed_extractor.dm

```diff
@@ -60,7 +84,7 @@ obj/machinery/seed_extractor/attackby(var/obj/item/O as obj, var/mob/user as mob
 		S.use(1)
 		new /obj/item/seeds/grassseed(loc)
 
-	if(O)
+	if(O) //not sure what this portion of code is supposed to do - needs some commenting

```

Comment:
If above O was deleted or used up on https://github.com/d3athrow/vgstation13/pull/3547/files#diff-211ff530d84779ca3328122cc4029712L60 it wouldnt want to try and continue to process.